src/components/Card.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./Card.css";
+import { Card as CardComp, Button } from "react-bootstrap";
+import { BsFillTrashFill } from "react-icons/bs";
+
+const Card = (props) => {
+  const onClickUpdateCard = () => {
+    const updateCard = {
+      ...props,
+      likesCount: props.likesCount + 1,
+    };
+    props.onUpdateLike(updateCard);
```

Comment:
Because `props` holds extra info like functions, the new object being passed to onUpdateLike is also passing those extra pieces of data to be held in the state object. Is this intended? 

I recommend passing back just the ID from here and having the ancestor who holds the state manage updating the like count over than having this component be responsible for knowing what shape the ancestor component stores its data in.